extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -163,4 +163,10 @@ export class AmpShareTracking extends AMP.BaseElement {
   }
 }
 
-AMP.registerElement('amp-share-tracking', AmpShareTracking);
+
+// Install the extension.
+// DO NOT SUBMIT: First, we must implement compiler rewrite path to unwrap
+// this code.
+AMP.extension('amp-share-tracking', () => {
```

Comment:
Ok, im trying to weight if we want to add this dynamically during dev (just through the wrapper configuration) or remove it during production build time (a bit more work but also do-able)